CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 - Create `PurpleCapsuleOfInfinityWidth` as `ButtonStyle`. Create `LoginView` and apply in the `WoofApp` to display when running the app when the user role is not selected - [#160](https://github.com/ios-course/ironfoudation-team-project/pull/160)
 - Add the `EditSitterInformationView` and the reusable element `TextEditorWithPlaceholder` - [#159](https://github.com/ios-course/ironfoudation-team-project/pull/159)
 - Add ability to change the user role using `Logout` button on the `SitterMainTabView` and `OwnerMainTabView` screens - [#167](https://github.com/ios-course/ironfoudation-team-project/pull/167)
+- Add `SitterProfileView` to display and edit personal information about the sitter - [#114](https://github.com/ios-course/ironfoudation-team-project/issues/114)
```

Comment:
It looks like you made the changes in the wrong place.
Please move these changes to the `Unreleased` section.

Reply:
Yes, thanks, fixed it.